code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -4006,3 +4017,12 @@ datum
 			mix_phrase = "The mixture comes together slowly. It doesn't seem like it wants to be here."
 			required_reagents = list("poor_cement" = 1, "silicon_dioxide" = 5, "water" = 1)
 			result_amount = 7
+
+		hemotoxin
```

Comment:
Looks like you might have accidentally made a branch from a pre-existing one

Reply:
yea im fixing it rn